{{cookiecutter.project_slug}}/Dockerfile_template

```diff
@@ -0,0 +1,12 @@
+FROM python:{PYTHON_VERSION}-slim
```

Comment:
Why not name this file as Dockerfile?

Reply:
Because the place holder `{PYTHON_VERSION}` will make this file an invalid Dockerfile, so I add a template postfix. But I think renaming it to `Dockerfile` is also feasible.

Reply:
I've integrated them together since it seems like there will be more variables to add from cookiecutter's settings.